dist/mapdc.js

```diff
@@ -64992,7 +64989,7 @@ function renderNominalLegend(state, dispatch) {
                     h_1.default("div.color", {
                         style: { background: state.range[index] }
                     }),
-                    h_1.default("div.text", "" + value)
+                    h_1.default("div.text", value)
```

Comment:
[Non-blocking comment] Out of curiosity, why the removal of `"" + value`? (Not saying that it's wrong, but just want to make sure that wasn't accidental. I believe that exists as an idiomatic way to ensure `value` is casted to a string, which may or may not be necessary if we know for sure `value` is always a string and/or `h_1.default()` doesn't mind if its arguments aren't strings.)

Reply:
I think this was unrelated code in the map legend, it was just not properly compiled. But `"" + whatever` is a bad way to cast something to a string. It's more readable and robust to do `String(value)` or even `value.toString()`, which would get done anyway since it seem to be assigning some text to a div text. Anyway.